banyand/measure/merger_test.go

```diff
@@ -28,9 +29,28 @@ import (
 
 	"github.com/apache/skywalking-banyandb/pkg/fs"
 	pbv1 "github.com/apache/skywalking-banyandb/pkg/pb/v1"
+	"github.com/apache/skywalking-banyandb/pkg/run"
 	"github.com/apache/skywalking-banyandb/pkg/test"
 )
 
+// nopProtector is a dummy Protector implementation for tests.
```

Comment:
Could you move it to the protector package? The stream and measure components could share it.